frontend/build.gradle

```diff
@@ -13,6 +13,7 @@ allprojects {
     version = '1.0'
 
     repositories {
+        mavenLocal()
```

Comment:
Please remove this. This should only be for local testing

Reply:
Done

Thanks, for pointing. It's my local change for testing.